packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -155,7 +155,8 @@ const CrudToolbar: React.FunctionComponent<CrudToolbarProps> = ({
   const controlId = useId();
   const prevButtonDisabled = useMemo(() => page === 0, [page]);
   const nextButtonDisabled = useMemo(
-    () => (count ? 20 * (page + 1) >= count : true),
+    // If we don't know the count, we can't know if there are more pages.
+    () => (count ? 20 * (page + 1) >= count : false),
```

Comment:
Should this be 

```suggestion
    () => (count !== undefined ? 20 * (page + 1) >= count : false),
```

or 

```suggestion
    () => (isNaN(count) ? 20 * (page + 1) >= count : false),
```

since `count === 0` would imply that there are no more pages to come?

Reply:
thanks! added this and null check